packages/next-plugin-sentry/src/on-error-server.js

```diff
@@ -1,5 +1,27 @@
-import * as Sentry from '@sentry/node'
+import { captureException, flush, Handlers, withScope } from '@sentry/node'
+import getConfig from 'next/config'
 
-export default async function onErrorServer({ err }) {
-  Sentry.captureException(err)
+const { parseRequest } = Handlers
+
+export default async function onErrorServer({ err, req }) {
+  const { serverRuntimeConfig = {}, publicRuntimeConfig = {} } =
+    getConfig() || {}
```

Comment:
`next/config` should not be used. This is not compatible with Next.js' SSG mode and only exists for legacy reasons.